### PR TITLE
:wrench: (feat) apply equal counts post test to anonymized configs

### PR DESCRIPTION
**Pull request by zoemcl**

### CODE REVIEW

File: dags/anonymized_cathydb.py
Comment (laurabegin) [inline dags/anonymized_cathydb.py:1]: À conserver

File: dags/config/red/curated/unic_config.json [hunk 1]
```diff
@@ -22,7 +22,7 @@
     "schemas": [],
     "datasets":
     [
-      {"dataset_id":"anonymized_unic_patient_index", "cluster_type": "medium", "run_type": "initial", "cluster_specs": {}, "dependencies": [], "pre_tests": [], "post_tests": []}
+      {"dataset_id":"anonymized_unic_patient_index", "cluster_type": "medium", "run_type": "initial", "cluster_specs": {}, "dependencies": [], "pre_tests": [], "post_tests": ["post_tests"]}
```
Comment (laurabegin) [inline dags/config/red/curated/unic_config.json:25]: equal_counts?

File: dags/config/red/ingestion/icca_config.json [hunk 2]
```diff
@@ -36,7 +36,7 @@
     "schemas": [],
     "datasets":
     [
-      {"dataset_id": "anonymized_icca_blood_gas"         , "cluster_type": "small" , "run_type": "default", "cluster_specs": {}, "dependencies": [], "pre_tests": [], "post_tests": []},
+      {"dataset_id": "anonymized_icca_blood_gas"         , "cluster_type": "small" , "run_type": "default", "cluster_specs": {}, "dependencies": [], "pre_tests": [], "post_tests": ["post_tests"]},
```
Comment (laurabegin) [inline dags/config/red/ingestion/icca_config.json:39]: equal_counts?

**Review by laurabegin — CHANGES_REQUESTED** [review]

File: dags/anonymized_cathydb.py
Comment (zoemcl) [reply dags/anonymized_cathydb.py:1]: oops je lai enlever au lien de anonymized_icca_htr!

**Pull request merged**